packages/flutter_web_plugins/lib/flutter_web_plugins.dart

```diff
@@ -0,0 +1,5 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+export 'src/plugin_registry.dart';
```

Comment:
Did you need to make any modifications to the package graph for this? Or is it essentially pulled in transitively via any plugins?

Reply:
It is assuming that if there are web plugins they must import `package:flutter_web_plugins`. Modifying the package graph is a more substantial change because the build daemon makes the package graph from the `pubspec.yaml`, so we would have to make our our daemon that watches for file changes and create the `PackageGraph` ourselves (like how we do for a single build).

Reply:
That makes sense, since you don't import flutter_web_plugins into the shell if there is no registrant it should work.